pykeepass/entry.py

```diff
@@ -125,14 +129,6 @@ def notes(self):
     def notes(self, value):
         return self._set_string_field('Notes', value)
 
-    @property
-    def icon(self):
```

Comment:
Why remove the `icon` getter and setters?

Reply:
because the exact same functions exist in the baseelement; didn't seem necessary